Sprint-3/quote-generator/index.html

```diff
@@ -3,13 +3,23 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Title here</title>
+    <link rel="stylesheet" href="style.css" />
+    <title>Quote Generator</title>
     <script defer src="quotes.js"></script>
```

Comment:
typically the js script goes to the bottom. Can you try and find out why? :)

Reply:
JS is usually placed at the bottom to prevent blocking the page load and ensure the DOM is fully loaded first. Here, I used defer, which achieves the same effect by delaying script execution until after the HTML is parsed.